tools/c7n_oci/tests/cassettes/TestInstance.test_instance_stop.yml

```diff
@@ -0,0 +1,4211 @@
+          resourceType: Instance
+          searchContext: null
+          systemTags: {}
+          timeCreated: '2023-03-26T06:57:22.575Z'
+        - additionalDetails: {}
+          availabilityDomain: NQNU:US-ASHBURN-AD-1
+          compartmentId: ocid1.compartment.oc1..<unique_ID>
+          definedTags:
+            Oracle-Tags:
+              CreatedBy: user@example.com
+              CreatedOn: '2023-03-26T06:57:06.145Z'
+          displayName: jumperhost
+          freeformTags:
+            eligible_for_resize_vv: 'true'
+            need_action: 'true'
+          identifier: ocid1.instance.oc1..<unique_ID>
+          identityContext: {}
+          lifecycleState: STOPPED
+          resourceType: Instance
+          searchContext: null
+          systemTags: {}
+          timeCreated: '2023-03-26T06:57:06.678Z'
+        - additionalDetails: {}
+          availabilityDomain: NQNU:US-ASHBURN-AD-2
+          compartmentId: ocid1.compartment.oc1..<unique_ID>
+          definedTags:
+            Oracle-Tags:
+              CreatedBy: oke
+              CreatedOn: '2023-03-10T18:08:00.409Z'
+          displayName: oke-c3ejqah2yqq-nzhvaorxrma-stsybdx6llq-2
+          freeformTags:
+            eligible_for_resize_vv: 'true'
+            need_action: 'true'
+          identifier: ocid1.instance.oc1..<unique_ID>
+          identityContext: {}
+          lifecycleState: STOPPED
+          resourceType: Instance
+          searchContext: null
+          systemTags: {}
+          timeCreated: '2023-03-10T18:08:05.012Z'
+        - additionalDetails: {}
+          availabilityDomain: NQNU:US-ASHBURN-AD-2
+          compartmentId: ocid1.compartment.oc1..<unique_ID>
+          definedTags:
+            Oracle-Tags:
+              CreatedBy: oke
+              CreatedOn: '2023-03-10T18:08:02.002Z'
+          displayName: oke-c3ejqah2yqq-nzhvaorxrma-stsybdx6llq-1
+          freeformTags:
+            eligible_for_resize_vv: 'true'
+            need_action: 'true'
+          identifier: ocid1.instance.oc1..<unique_ID>
+          identityContext: {}
+          lifecycleState: STOPPED
+          resourceType: Instance
+          searchContext: null
+          systemTags: {}
+          timeCreated: '2023-03-10T18:08:02.955Z'
+        - additionalDetails: {}
+          availabilityDomain: NQNU:US-ASHBURN-AD-2
+          compartmentId: ocid1.compartment.oc1..<unique_ID>
+          definedTags:
+            Oracle-Tags:
+              CreatedBy: oke
+              CreatedOn: '2023-03-10T18:08:00.409Z'
+          displayName: oke-c3ejqah2yqq-nzhvaorxrma-stsybdx6llq-0
+          freeformTags:
+            eligible_for_resize_vv: 'true'
+            need_action: 'true'
+          identifier: ocid1.instance.oc1..<unique_ID>
+          identityContext: {}
+          lifecycleState: STOPPED
+          resourceType: Instance
+          searchContext: null
+          systemTags: {}
+          timeCreated: '2023-03-10T18:08:00.975Z'
+        - additionalDetails: {}
```

Comment:
does seem like we have a some extraneous data here versus what's needed for the test.